### DISCUSSION TITLE
Add new tags/Correspondents/Types from Document multi-select

### DISCUSSION THREAD

**Discussion by jeremymeyers** [jonaswinkler/paperless-ng#836] [Feature Requests]
In documents view, if i select multiple documents, i can assign different tags/correspondents/document types from the dropdowns, but i can only select from pre-existing tag/correspondent/types.   It would be great to be able to type a new one of any of those into their associated dropdown and have a "create '[tagname] and assign'" option.

**Comment by jpbaril** [comment]
Yep. I was going to create a new ticket for this feature then aw this discussion.
It would really be the same feature that can be seen when editing a document, you can either assign an existing tag or create a new one.
